include/NeoFOAM/timeIntegration/rungeKutta.hpp

```diff
@@ -0,0 +1,271 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2023 NeoFOAM authors
+
+#pragma once
+
+#include <functional>
+#include <memory>
+
+#include "NeoFOAM/fields/field.hpp"
+#include "NeoFOAM/timeIntegration/timeIntegration.hpp"
+#include "NeoFOAM/timeIntegration/sundials.hpp"
+
+
+// Where to put?
+struct NFData
```

Comment:
```suggestion
struct RKData
```
or 
```suggestion
struct SundialsData
```
maybe a better name?